portfolio/src/main/webapp/index.html

```diff
@@ -2,17 +2,69 @@
+            </div>
+        </div>
+    </header>
+    <main>
+        <div class="card bg-white text-dark border-white">
+            <div class="card-body">
+                <h1 class="card-text">About Me</h1>
+                <h2 class="card-text">College</h2>
+                <p class="card-text">I am a rising junior at Santa Clara University majoring in Computer Science and Engineering.  As a campus ambassador for Google, I organized events on campus, introducing fellow students to Google products.</p>
+                <h3 class="card-text">Google Cloud Platform Student Innovator</h3>
+                <p class="card-text">As the first Google Cloud Platform Student Innovator at my university, I hosted Google Cloud Platform workshops, tabled at annual hackathons as a mentor, and gave class presentations to undergraduate and graduate students.</p>
+                <img src="images/Google Cloud Platform Student Innovator.jpeg" width="250" />
+                <h3 class="card-text">Google Developer Student Club Lead</h3>
+                <p class="card-text">As a Google Developer Student Club Lead, I was invited to attend Developer Student Clubs United States Leads Summit 2019.</p>
+                <img src="images/Google Developer Student Club Lead.jpeg" width="250" />
+                <h2 class="card-text">High School</h2>
+                <p class="card-text">I was born and raised in the Bay Area, and I attended Palo Alto High School.  In high school, I founded two local clubs, Coding Trek and Silicon Valley Star Trek Club, and I was featured in four news articles.</p>
+                <h3 class="card-text">Coding Trek</h3>
+                <h4 id="text" class="card-text"></h4>
+                <p class="card-text">As the founder of Coding Trek, I designed a computer science curriculum for beginners and organized a series of free coding workshops for all ages.  My club was featured in <a href="https://www.losaltosonline.com/news/sections/schools/213-school-noteworthies/56854-" target="_blank">Los Altos Town Crier</a>.</p>
+                <h3 class="card-text">Silicon Valley Star Trek Club</h3>
+                <marquee><h4 class="card-text">To Boldly Go Where No One Has Gone Before</h4></marquee>
+                <p class="card-text">As the founder of Silicon Valley Star Trek Club, I organized weekly discussions about artificial intelligence, utopia, diversity, equality, justice, humanity, and technology.  I also collected and compiled 300 Star Trek quotes.  My club was featured in <a href="http://www.mercurynews.com/2017/04/26/star-trek-travels-well-even-to-the-next-generation/" target="_blank">The Mercury News</a>, <a href="https://www.paloaltoonline.com/news/2017/04/22/palo-alto-teen-creates-her-own-trekkie-starship-crew" target="_blank">Palo Alto Online</a>, and <a href="http://www.losaltosonline.com/news/sections/community/177-features/54219-teen-hopes-star-trek-club-lives-long-and-prospers" target="_blank">Los Altos Town Crier</a>.</p>
```

Comment:
Remind me to show you my extensive collection of Eaglemoss starship models sometime.  😁   

I also might have written a web component HTML and JavaScript library for making LCARS interfaces...